{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/contact/views.py

```diff
@@ -16,35 +14,17 @@ class ContactView(CreateView):
 
     def form_valid(self, form):
         context = {
+            'reply_to': form.cleaned_data['email'],
             'first_name': form.cleaned_data['first_name'],
             'last_name': form.cleaned_data['last_name'],
             'phone_number': form.cleaned_data['phone_number'],
             'reason_for_enquiry': form.cleaned_data['reason_for_enquiry'],
             'email': form.cleaned_data['email'],
             'job_title': form.cleaned_data['job_title'],
             'message': form.cleaned_data['message'],
-            'settings': settings,
```

Comment:
Is it not useful to have settings in the template context? e.g. `{{ settings.SITE_NAME }}`

Reply:
Perhaps, but maybe it should be made available globally rather than just for the contact template?